instrumentation/netty/src/main/java/brave/netty/HttpNettyAdapter.java

```diff
@@ -0,0 +1,31 @@
+package brave.netty;
+
+import brave.http.HttpServerAdapter;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpResponse;
+
+class HttpNettyAdapter extends HttpServerAdapter<HttpRequest, HttpResponse> {
+  @Override
+  public String method(HttpRequest request) {
+    return request.method().name();
+  }
+
+  @Override
+  public String url(HttpRequest request) {
+    StringBuffer url = new StringBuffer("http://");
+    String host = request.headers().get("HOST");

```

Comment:
hmm maybe comment that this does not return the url unless the host header is present (even if it is likely)